scripts/util/keystore-setup

```diff
@@ -22,55 +22,78 @@ KS_PASS="$3"
 KS_VALIDITY="$4"
 CSR_FILE="$5"
 
+LOG_FILE="/var/log/cloudstack/agent/keystore-setup.log"
+
 ALIAS="cloud"
 LIBVIRTD_FILE="/etc/libvirt/libvirtd.conf"
 
+log() {
```

Comment:
I am not sure if it installed by default on your host's OS, but there is a tool called logger to do this, no need for a script function.